apps/backoffice-v2/src/common/components/molecules/DataTable/DataTable.tsx

```diff
@@ -0,0 +1,309 @@
+                        'sticky top-0 z-10 h-[34px] bg-white p-1 text-[14px] font-bold text-[#787981]',
+                        {
+                          '!pl-3.5': index === 0,
+                        },
+                        props?.head?.className,
+                      )}
+                    >
+                      {header.column.id === 'select' && (
+                        <span className={'pe-4'}>
+                          {flexRender(header.column.columnDef.header, header.getContext())}
+                        </span>
+                      )}
+                      {enableSorting && header.column.id !== 'select' && (
+                        <button
+                          className="flex h-9 flex-row items-center gap-x-2 px-3 text-left text-[#A3A3A3]"
+                          onClick={() => header.column.toggleSorting()}
+                        >
+                          <span>
+                            {flexRender(header.column.columnDef.header, header.getContext())}
+                          </span>
+                          <ChevronDown
+                            className={ctw('d-4', {
+                              'rotate-180': header.column.getIsSorted() === 'asc',
+                            })}
+                          />
+                        </button>
+                      )}
+                      {!enableSorting &&
+                        !header.isPlaceholder &&
+                        flexRender(header.column.columnDef.header, header.getContext())}
+                    </TableHead>
+                  );
+                })}
+              </TableRow>
+            ))}
+          </TableHeader>
+          <TableBody {...props?.body}>
+            {!!table.getRowModel().rows?.length &&
+              table.getRowModel().rows?.map(row => (
+                <React.Fragment key={row.id}>
+                  <TableRow
+                    key={row.id}
+                    {...props?.row}
+                    className={ctw(
+                      'h-[76px] border-b-0 even:bg-[#F4F6FD]/50 hover:bg-[#F4F6FD]/90',
+                      props?.row?.className,
+                    )}
+                  >
+                    {row.getVisibleCells()?.map(cell => (
+                      <TableCell
+                        key={cell.id}
+                        {...props?.cell}
+                        className={ctw('!py-px !pl-3.5', props?.cell?.className)}
+                      >
+                        {CellWrapper ? (
+                          <CellWrapper cell={cell}>
+                            {flexRender(cell.column.columnDef.cell, cell.getContext())}
+                          </CellWrapper>
+                        ) : (
+                          flexRender(cell.column.columnDef.cell, cell.getContext())
+                        )}
+                      </TableCell>
+                    ))}
+                  </TableRow>
+                  {CollapsibleComponent && (
```

Comment:
Thoughts with the alias below?
```suggestion
                  {CollapsibleContent && (
```